specification/communication/data-plane/Rooms/preview/2023-01-01-preview/communicationservicesrooms.json

```diff
@@ -515,7 +515,6 @@
       "description": "The endpoint of the Azure Communication resource.",
       "required": true,
       "type": "string",
-      "format": "uri",
```

Comment:
Removing the format for several reasons: 
- aligning with all other ACS services - none of their endpoint parameter has format
- the format property is not supported as part of Swashbuckle